docs/normalization-rules.rst

```diff
@@ -90,6 +90,14 @@ string, it points to a :doc:`custom method <customize>`.
    >>> v.errors
    {'a': ["default value for 'a' cannot be set: Circular dependencies of default setters."]}
 
+You can even use both `default` and `readonly` on the same field. This will
```

Comment:
I will look into that. First I finally have to setup Sphinx to compile the docs locally ;).